common/cpp/react/renderer/components/rnsvg/RNSVGConcreteShadowNode.h

```diff
@@ -0,0 +1,70 @@
+#pragma once
+
+#include <react/renderer/components/view/ViewEventEmitter.h>
+#include <react/renderer/components/view/ViewProps.h>
+#include <react/renderer/core/ConcreteShadowNode.h>
+#include "RNSVGLayoutableShadowNode.h"
+
+namespace facebook::react {
+
+template <const char *concreteComponentName, typename PropsT = ViewProps>
+class RNSVGConcreteShadowNode : public ConcreteShadowNode<
+                                    concreteComponentName,
+                                    RNSVGLayoutableShadowNode,
+                                    PropsT,
+                                    ViewEventEmitter,
+                                    StateData,
+                                    false> {
+ public:
+  using BaseShadowNode = ConcreteShadowNode<
+      concreteComponentName,
+      RNSVGLayoutableShadowNode,
+      PropsT,
+      ViewEventEmitter,
+      StateData,
+      false>;
+
+  RNSVGConcreteShadowNode(
+      const ShadowNodeFragment &fragment,
+      const ShadowNodeFamily::Shared &family,
+      ShadowNodeTraits traits)
+      : BaseShadowNode(fragment, family, traits) {}
+
+  RNSVGConcreteShadowNode(
+      const ShadowNode &sourceShadowNode,
+      const ShadowNodeFragment &fragment)
+      : BaseShadowNode(sourceShadowNode, fragment) {}
```

Comment:
I think these constructors aren't needed when you have `using BaseShadowNode::BaseShadowNode;` below.

https://en.wikipedia.org/wiki/C%2B%2B11#Object_construction_improvement